app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -77,11 +77,14 @@ protected void onCreate(Bundle savedInstanceState) {
         View view = binding.getRoot();
         setContentView(view);
         // Initialize the Voxeet SDK
-        // WARNING: It is best practice to use the VoxeetSDK.initialize function with an Access Token to initialize the SDK.
         // Please read the documentation at:
         // https://docs.dolby.io/interactivity/docs/initializing
-        throw new IllegalStateException("<---- Remove this line and set your keys below to use this sample !!");
-        VoxeetSDK.initialize("", "");
+        // Grab your demos client access token from your dashboard and insert into accessToken variable
```

Comment:
Update to // Generate a test client access token from the Dolby.io dashboard and insert into accessToken variable